.travis.yml

```diff
@@ -18,5 +18,7 @@ rvm:
   - 1.9.3
   - "2.0"
   - "2.1"
+  - "2.2"
+  - "2.3"
```

Comment:
Ugh, it failed on 2.4 because RVM doesn't seem to want to fuzzy match. We may have to be more specific for 2.4.